src/test/java/pl/printo3d/onedcutter/cutter1d/cutter/services/OrderServiceTest.java

```diff
@@ -128,12 +135,35 @@ public void makeFreeOrder_should_return_default_vals()
     public void saveActiveOrder_should_niewiemco() // toć void, wiec coz ma zwrocic? @_@
     {
         // given
+        UserModel principal = new UserModel(
+                "username",
+                "password"
+        );
+        OrderModel activeOrderModel = new OrderModel();
+        CutOptions cutOptions = new CutOptions();
+        cutOptions.setId(CUT_OPTIONS_ID);
+        activeOrderModel.setCutOptions(cutOptions);
+        principal.setActiveOrderModel(activeOrderModel);
+        when(userService.loadUserByUsername(USERNAME))
+                .thenReturn(principal);
+
         OrderModel testOrder = new OrderModel();
+        testOrder.setCutOptions(new CutOptions());
+        List<CutModel> cutList = Collections.singletonList(new CutModel());
+        testOrder.setCutList(cutList);
+        List<StockModel> stockList = Collections.singletonList(new StockModel());
+        testOrder.setStockList(stockList);
+        testOrder.setProjectName(PROJECT_NAME);
 
         // when
-        //orderServiceTest.saveActiveOrder(testOrder);
+        orderServiceTest.saveActiveOrder(testOrder);
 
         // then
+        assertEquals(cutList, principal.getActiveOrderModel().getCutList());
```

Comment:
do metody przekazujesz orfderModel w parametrze i wewnątrz tej metody coś się dzieje z tym obiektem. Jakies pola na nim są ustawiane. Dokładnie to można sprawdzić w tescie. Upewniamy się że pola które są ustawiane dostają wartości jakich się spodziewamy